doc/index.md

```diff
@@ -1,10 +1,55 @@
+[myget-npgsql]:          https://www.myget.org/feed/npgsql/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL
+[myget-npgsql-unstable]: https://www.myget.org/feed/npgsql-unstable/package/nuget/Npgsql.EntityFrameworkCore.PostgreSQL
+
+[* Branches]: <>
+[tree-dev]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/dev
+[tree-master]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/master
+[tree-hotfix/2.2.*]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/hotfix/2.2.6
+[tree-hotfix/2.1.*]: https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/tree/hotfix/2.1.11
+
+[* Azure Pipelines (badges)]: <>
+[build-dev]:          https://dev.azure.com/npgsql/EFCore.PG/_apis/build/status/npgsql.Npgsql.EntityFrameworkCore.PostgreSQL?branchName=dev
+[build-master]:       https://dev.azure.com/npgsql/EFCore.PG/_apis/build/status/npgsql.Npgsql.EntityFrameworkCore.PostgreSQL?branchName=master
+[build-hotfix/2.2.*]: https://dev.azure.com/npgsql/EFCore.PG/_apis/build/status/npgsql.Npgsql.EntityFrameworkCore.PostgreSQL?branchName=hotfix/2.2.6
+[build-hotfix/2.1.*]: https://dev.azure.com/npgsql/EFCore.PG/_apis/build/status/npgsql.Npgsql.EntityFrameworkCore.PostgreSQL?branchName=hotfix/2.1.11
+
+[* Azure Pipelines (links)]: <>
+[ci-dev]:          https://dev.azure.com/npgsql/EFCore.PG/_build/latest?branchName=dev
+[ci-master]:       https://dev.azure.com/npgsql/EFCore.PG/_build/latest?branchName=master
+[ci-hotfix/2.2.*]: https://dev.azure.com/npgsql/EFCore.PG/_build/latest?branchName=hotfix/2.2.6
+[ci-hotfix/2.1.*]: https://dev.azure.com/npgsql/EFCore.PG/_build/latest?branchName=hotfix/2.1.11
+
+[* Dynamic Versions]: <>
+[nuget-v]:              https://img.shields.io/nuget/v/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=nuget
+[nuget-vpre]:           https://img.shields.io/nuget/vpre/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=nuget
+[npgsql-v]:             https://img.shields.io/myget/npgsql/v/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=npgsql
+[npgsql-vpre]:          https://img.shields.io/myget/npgsql/vpre/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=npgsql
+[npgsql-unstable-v]:    https://img.shields.io/myget/npgsql-unstable/v/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=npgsql-unstable
+[npgsql-unstable-vpre]: https://img.shields.io/myget/npgsql-unstable/vpre/Npgsql.EntityFrameworkCore.PostgreSQL.svg?label=npgsql-unstable
+
+[* Static Versions]: <>
+[nuget-2.2.*]:           https://img.shields.io/badge/nuget-v2.2.*-blue.svg
+[nuget-2.1.*]:           https://img.shields.io/badge/nuget-v2.1.*-blue.svg
+[npgsql-2.2.*]:          https://img.shields.io/badge/npgsql-v2.2.*-yellow.svg
+[npgsql-2.1.*]:          https://img.shields.io/badge/npgsql-v2.1.*-yellow.svg
+[npgsql-unstable-2.2.*]: https://img.shields.io/badge/npgsql--unstable-v2.2.*-yellow.svg
+[npgsql-unstable-2.1.*]: https://img.shields.io/badge/npgsql--unstable-v2.1.*-yellow.svg
+
+
```

Comment:
Do we really need this in docs?

Reply:
My primary interest was to update the readme, but since we already had the old badges in the docs, I wanted to keep the two in sync.

Reply:
I tend to agree with @YohDeadfall - it seems important for the docs home page to be very concise with a clear code sample, rather than listing all this stuff... Maybe a separate page and a link to it?

Reply:
Sure, might even be able to just link the readme into the table of contents? That would solve (one of) the synchronization problems.

Reply:
It's a bit weird for a TOC entry to point outside of the site... But a separate page for this on the site is probably pretty easy to do, no?